packages/beacon-node/src/api/impl/lightclient/index.ts

```diff
@@ -12,42 +12,55 @@ export function getLightclientApi({
 }: Pick<ApiModules, "chain" | "config">): ApplicationMethods<routes.lightclient.Endpoints> {
   return {
     async getLightClientUpdatesByRange({startPeriod, count}) {
+      const lightClientServer = chain.lightClientServer;
```

Comment:
could just make this assignment before the `return`, and then just do the assert at the begging of the function